src/routes.js

```diff
@@ -11,6 +11,7 @@ import { RestrictedRoute } from "./Auth/AuthRoutes/RestrictedRoutes";
 
 import { HomePage } from "./Pages/Home/home-page";
 import { Notes } from "./Pages/Notes/notes";
```

Comment:
you may use index.js file in the same directory for import export.
Like:  import {HomePage, Notes} from './Pages/index.js';